scripts/collector/mysql/sql/users.sql

```diff
@@ -0,0 +1,37 @@
+with src as (
```

Comment:
We need an alternative for `mysql.users` if we ever need to include Maria DB support.